<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MockUI</class>
 <widget class="QMainWindow" name="MockUI">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentCU Mock UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 170, 218);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>771</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 40pt &quot;Bahnschrift&quot;;</string>
     </property>
     <property name="text">
      <string>Homing Procedure in process</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>191</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../Downloads/home.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="IE_frame_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>571</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 215, 209);
border-style: none;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>411</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="rehome_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(255, 105, 97);
font: 14pt &quot;Bahnschrift&quot;;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
         </property>
         <property name="text">
          <string>Rehome</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="start_ventilation_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(154, 202, 123);
font: 14pt &quot;Bahnschrift&quot;;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;</string>
         </property>
         <property name="text">
          <string>Confirm</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Bahnschrift&quot;;
border-style: none;</string>
      </property>
      <property name="text">
       <string>Is this correct?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>401</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Bahnschrift&quot;;
border-style: none;</string>
         </property>
         <property name="text">
          <string>Ambu Bag Diameter : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bag_size_label">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;Bahnschrift&quot;;
border-style: none;</string>
         </property>
         <property name="text">
          <string>8''</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
